### PR TITLE
Overview change

### PR DESCRIPTION
**Pull request by niksseif**
Description 
I have change the overview of this README, and ...

Changes :  
 - [] Add overview

Question : 
Hey team I have left few areas as I was not sure if the overview you are expecting is right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Merge Conflicts Practice
 
 ## Overview
-
+- Hello this overview is from Nik - HAHAHAHAH
```
Comment (Kalikoze) [inline README.md:4]: Love that we're adding some additional context here.

**Review by Kalikoze — CHANGES_REQUESTED** [review]
Overall, I love the changes that have been added to the README.  Maybe we can remove the "HAHAHAHA"

**Comment by niksseif** [comment]
Thanks Travis for your recommendation, I have resolved this in [7492762](https://github.com/Kalikoze/merge-conflicts-practice/pull/1/commits/7492762488f1a84609b5990d65c0545e702b4db5)

**Review by Kalikoze — APPROVED** [review]
Thanks so much for making those changes, let's ship it! 🚢

**Pull request merged**